apis/wikipedia.py

```diff
@@ -18,92 +27,73 @@ def retrieve_short_description(data: str) -> str:
 
 
 def get_short_description(title: str) -> Dict:
+    try:
+        api_data = {}
 
-    api_data = {}
-
-    PARAMS = {
-    "action": "query",
-    "titles": title,
-    "prop": "pageprops",
-    "format": "json"
-    }
+        PARAMS = {
+        "action": "query",
+        "titles": title,
+        "prop": "pageprops",
+        "format": "json"
+        }
 
-    DATA = requests.get(url=wikipedia_api_url, params=PARAMS).json()
-    api_data['short description'] = retrieve_short_description(DATA)
+        DATA = requests.get(url=wikipedia_api_url, params=PARAMS).json()
+        api_data['short description'] = retrieve_short_description(DATA)
 
-    return api_data
+        return api_data
+    
+    except Exception as e:
+        return {'message': 'error'}

```

Comment:
Have a more specific error message here 
```suggestion
		  return {'message': '[wikipedia] error getting short description'}
```